dev/benchmarks/macrobenchmarks/lib/src/web/bench_harness.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter/material.dart';
+
+import 'recorder.dart';
+
+class BenchWidgetRecorder extends WidgetRecorder {
+  BenchWidgetRecorder() : super(name: benchmarkName);
+
+  static const String benchmarkName = 'bench_widget_recorder';
+
+  @override
+  Widget createWidget() {
+    // This is intentionally using a simple widget. The benchmark is meant to
+    // measure the overhead of the harness, so this method should induce as
+    // little work as possible.
+    return Container();
```

Comment:
Tiny nit: maybe use `const SizedBox.expand()`? That's what we use here: https://github.com/flutter/flutter/blob/ee7788cf4c03788837639101fae8dd87cf87579d/dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart#L600-L602

and in other non-web benchmarks too: https://github.com/flutter/flutter/blob/0283d8a8d0d807ad9055b189be8fda11abe90238/dev/benchmarks/macrobenchmarks/lib/src/animated_blur_backdrop_filter.dart#L58-L61

Reply:
Done.